devops/Makefile

```diff
@@ -192,6 +192,7 @@ init-secrets:
 		-p ZEEBE_CLIENT_SECRET=$(ZEEBE_CLIENT_SECRET) \
 		-p CAMUNDA_OAUTH_URL=$(CAMUNDA_OAUTH_URL) \
 		-p ZEEBE_GRPC_WORKER_LONGPOLL_SECONDS=$(ZEEBE_GRPC_WORKER_LONGPOLL_SECONDS) \
+		-p QUEUE_DASHBOARD_TOKEN_SECRET=$(QUEUE_DASHBOARD_TOKEN_SECRET) \
```

Comment:
Init secrets is missing `${QUEUE_DASHBOARD_TOKEN_EXPIRATION_SECONDS}`

Reply:
It is not missing, it is just configured as a default value. I was not sure about exposing it or not.
I can add to the GitHub actions.

Reply:
`QUEUE_DASHBOARD_TOKEN_EXPIRATION_SECONDS` was changed to be exposed in GitHub vars and, if not provided, will be set at 3600 by the `make`.